### DISCUSSION TITLE
Nature Shader Essentials for NR 6

### DISCUSSION THREAD

**Discussion by fullerfusion** [visualdesigncafe/nature-renderer#85] [Help]
I'm confused as to why Nature Shader Essentials was removed from the NR 6 version? Will this be returning in a future update or will I have to purchase this separately in the future? 

I upgraded my project from 2022LTS to Unity 6 and now the shaders from NR 2022 are no longer available. My materials are all pink in the scene. These were very crucial shaders we were using for our project. Hopefully I can get a quick response from someone to help me with this situation.

**Comment by maxplooi** [comment]
Unity made a lot of significant changes to the render pipelines which caused the Nature Shaders to break. I felt that it was not helpful to constantly delay Nature Renderer updates because the Nature Shaders are not working, even though Nature Renderer itself works perfectly. It was hurting all the developers who are using Nature Renderer with their own shaders. So, I decided to no longer bundle the shaders with Nature Renderer and keep them as a separate asset. 

Once I figure out how to get the shaders compatible with Unity 6 there will be a new release of the shaders. No idea how long it will take though, updating these shaders requires completely reverse-engineering the render pipelines in Unity since Unity does not provide any documentation on their shaders.

**Reply by insthync** [reply]
So, essentials shader from the older version can't be used in Nature renderer 6, we have to use the demo ones instead, right?
